### ISSUE TITLE
Add support for `fr` unit from CSS Grid spec

### ISSUE THREAD

**Issue by sgomes** [stylelint/stylelint#2307]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

The `unit-no-unknown` rule doesn't know about the `fr` unit introduced in the [CSS Grid spec](https://www.w3.org/TR/css-grid-1/)

> Which rule, if any, is this issue related to?

`unit-no-unknown`

> What CSS is needed to reproduce this issue?

```css
.foo {
  display: grid;
  grid-template-columns: repeat(12, 1fr);
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "unit-no-unknown": true
  }
}
```

> Which version of stylelint are you using?

`7.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint packages/**/*.scss`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No, it's related to standard CSS, albeit a relatively new feature which is only starting to ship in browsers.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warning was flagged:

```shell
packages/mdc-layout-grid/_mixins.scss
 88:91  ✖  Unexpected unknown unit "fr"   unit-no-unknown
```

**Comment by jeddy3** [comment]
@sgomes Thanks for the report and for using the template.

I'll quickly add this in so it's included in the next release.

**Comment by sgomes** [comment]
Thanks for the quick reply, @jeddy3!
